src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Link} from '../Link/Link';
+import {HeaderBlock} from './styled';
+
+export const Header = () => {
+    return (
+        <HeaderBlock>

```

Comment:
please rename `HeaderBlock `to `HomePageHeader` (to mark that this component used only for HomePage)